src/shared/api/api.js

```diff
@@ -58,3 +58,28 @@ export const getByUrl = async (url, params, cancel, reloadOnUnauthorized) => {
     getWithToken(url, params, cancel, token, reloadOnUnauthorized)
   );
 };
+
+const getFormData = data => {
```

Comment:
IE 11 needs a polyfill for the `FormData` API

Reply:
thanks!